tests/unit-test-client.c

```diff
@@ -146,12 +153,14 @@ int main(int argc, char *argv[])
 
     /* Multiple bits */
     {
-        uint8_t tab_value[UT_BITS_NB];
+        uint8_t* tab_value = (uint8_t*)malloc(UT_BITS_NB);
```

Comment:
Do not cast to `(uint8_t *)` here, `void *` is implicitly casted to any typed pointer in C.

Reply:
I forgot, that only C++ enforces these casts. However all other malloc calls in htis file are casted this way, so ... leave it this way?

Reply:
I saw that as well. I know that Visual Studio 2006 or 2008 needed these casts even in C. This might be the reason for them being there.

Maybe in the future a clean up patch removing these might be better. So yes, you're right, leave them.